t/chronicle.t

```diff
@@ -0,0 +1,40 @@
+use strict;
+use warnings;
+
+use Test::More tests => 9;
+use Test::NoWarnings;
+use Data::Chronicle::Mock;
+use Date::Utility;
+
+my $d = { sample1 => [1, 2, 3],
+          sample2 => [4, 5, 6],
+          sample3 => [7, 8, 9] };
+
+my $d_old = { sample1 => [2, 3, 5],
+          sample2 => [6, 6, 14],
+          sample3 => [9, 12, 13] };
+
+my $first_save_epoch = time;
+
+my ($chronicle_r, $chronicle_w) = Data::Chronicle::Mock::get_mocked_chronicle();
+
+is $chronicle_w->set("vol_surface", "frxUSDJPY", $d), 1, "data is stored without problem";
+is_deeply $chronicle_r->get("vol_surface", "frxUSDJPY"), $d, "data retrieval works";
+is_deeply $chronicle_r->cache_reader->get("vol_surface::frxUSDJPY"), JSON::to_json($d), "redis has stored correct data";
+
+is $chronicle_w->set("vol_surface", "frxUSDJPY-old", $d_old, Date::Utility->new(0)), 1, "data is stored without problem when specifying recorded date";
```

Comment:
remove mention of vol surfaces; use a more understandable example test, that any member of the Perl public community can understand